CRM/Upgrade/Incremental/php/FiveFortyOne.php

```diff
@@ -83,6 +83,33 @@ public function upgrade_5_41_alpha1($rev) {
     $this->addTask('Replace contribution page id token in action schedule',
       'updateActionScheduleToken', 'contribution.contribution_page_id', 'contribution.contribution_page_id:label', $rev
     );
+    $this->addTask('Ensure multilingual installs have a default financial type for the membership price set.',
```

Comment:
Minor notes:
* It's not just multilingual it's any non-english.
* Historically these upgrade scripts haven't been consistent but 5.41 is currently beta1 so if we were being picky this would go in the beta1 upgrade function.

Reply:
Fixed